wsapi.go

```diff
@@ -321,6 +321,35 @@ func (s *Session) heartbeat(wsConn *websocket.Conn, listening <-chan interface{}
 	}
 }
 
+// SubscribeGuildData is used for SubscribeGuild()
+type SubscribeGuildData struct {
+	GuildID    string `json:"guild_id"`
+	Typing     bool   `json:"typing"`
+	Activities bool   `json:"activities"`
+}
+
+type subscribeGuildOp struct {
+	Op   int                `json:"op"`
+	Data SubscribeGuildData `json:"d"`
+}
+
+// SubscribeGuild event is needed to sent if you want TypingStart and other
```

Comment:
[Nit] Minor typo correction:

`// SubscribeGuild event needs to be sent sent if you want TypingStart and other`

Reply:
Correction of the correction 😄:

`// SubscribeGuild event needs to be sent if you want TypingStart and other`